src/routes/help/components.svelte

```diff
@@ -0,0 +1,238 @@
+<script>
+	import components from '../components/components.json';
+
+	const allTags = Array.from(new Set(components.map((item) => item.tags).flat()));
+	const tagItems = allTags.map((tag) => ({ label: tag, value: tag })).sort(ascending);
+	const allCategories = Array.from(new Set(components.map((item) => item.category).flat()));
+	const categoryItems = allCategories
+		.filter((cat) => cat !== '')
+		.map((cat) => ({ label: cat, value: cat }))
+		.sort(ascending);
+
+	let clipboardCopy = false;
+	const copyToClipboard = (text) => {
+		navigator.clipboard
+			.writeText(text)
+			.then(() => {
+				clipboardCopy = true;
+				setTimeout(() => (clipboardCopy = false), 1000);
+			})
+			.catch(() => alert('Clipboard copy Permission denied'));
+	};
+
+	let title = 'svelte-calender';
+	let url = 'https://github.com/6eDesign/svelte-calendar';
+	let description = 'A lightweight date picker with neat animations and a unique UX';
+	let npm = 'svelte-calender';
+	let image = '';
+	let category = 'Forms & User Input';
+	let tags = ['components and libraries', 'time and date'];
+	let addedOn = '';
+
+	$: jsonSnippet = {
+		title,
+		url,
+		description,
+		npm,
+		tags,
+		image,
+		addedOn,
+		category
+	};
+
+	function ascending(a, b) {
+		return a.label > b.label ? 1 : -1;
+	}
+</script>
+
+<h1>How to submit a new component?</h1>
+<p>
+	To add a new component on the website, the process is rather simple. You have to add your
+	component in <a
+		href="https://github.com/svelte-society/sveltesociety.dev/blob/master/src/routes/components/components.json"
+	>
+		components.json
+	</a>
+</p>
+
+<h2>Forking the project</h2>
+<p>
+	You can fork the <a href="https://github.com/svelte-society/sveltesociety.dev/">
+		GitHub project
+	</a>, add your component and then propose a Pull request.
+</p>
+
+<h2>Edit the file</h2>
+<p>
+	You can edit and propose your changes <a
+		href="https://github.com/svelte-society/sveltesociety.dev/edit/master/src/pages/components/components.json"
+	>
+		directly in GitHub
+	</a>
+</p>
+
+<h2>What information should I give ?</h2>
+<p>
+	Each component is represented by a JSON Object. Use the generator below to generate the JSON
```

Comment:
Please keep "Object" on this line and don't manually wrap the text. Text wrapping can be done by VS Code or whatever editor is being used to edit this file

Reply:
I'm indeed using VS Code. I don't know what is causing this hard wrap on format. `printWidth` is 100 inside `.prettierrc`, so that hard wrap shouldn't happen. For now, I replaced _JSON Object -> Object_. Still conveys the meaning I think. Please let me know how to fix this If you have any idea.